src/puppetlabs/trapperkeeper/bootstrap.clj

```diff
@@ -97,27 +99,28 @@
                            (string/join "\n" config-files)))
         config-files))))
 
-(schema/defn ^:private config-from-cwd :- (schema/maybe BootstrapFiles)
+(schema/defn ^:private config-from-cwd :- (schema/maybe [schema/Str])
```

Comment:
Woah, I did not know that. I'm guessing it's because `nil` can be passed into functions that take seqs? Good to know